x-pack/plugins/maps/public/classes/sources/source.ts

```diff
@@ -59,6 +59,9 @@ export interface ISource {
   isTimeAware(): Promise<boolean>;
   getImmutableProperties(dataFilters: DataFilters): Promise<ImmutableSourceProperty[]>;
   getAttributionProvider(): (() => Promise<Attribution[]>) | null;
+  /*
+   * Returns true when source implements IESSource interface
```

Comment:
I wonder if, at some future point, we should rename the interface to be more like `IESDslSource` to better distinguish it from ES|QL? Probably not worth it for this PR though.